mitmproxy/options.py

```diff
@@ -30,19 +30,19 @@ def __init__(
             app_port: int = APP_PORT,
             anticache: bool = False,
             anticomp: bool = False,
-            client_replay: Sequence[str] = (),
+            client_replay: Sequence[str] = [],
```

Comment:
There is an argument to be had that they should be tuples all along to prevent an `options.scripts.append(...)`, which would not trigger the change signals.